templates/components/itilobject/selfservice.html.twig

```diff
@@ -195,7 +195,7 @@
 
                      {{ include('components/itilobject/actors/main.html.twig', {
                         'item': item,
-                        'entities_id': params['entities_id'],
+                        'entities_id': entities_id,
```

Comment:
Line 250 should be updated too, I guess.

Reply:
In fact, modification was not required, as variable is defined from `'entities_id' => $options['entities_id'],` but now the source of value is more clear.